packages/fether-react/config-overrides.js

```diff
@@ -3,7 +3,9 @@ const { injectBabelPlugin } = require('react-app-rewired');
 /* config-overrides.js */
 module.exports = function override (config) {
   // use the MobX rewire to use @decorators
-  config = injectBabelPlugin('transform-decorators-legacy', config);
-
+  config = injectBabelPlugin(
```

Comment:
https://github.com/timarney/react-app-rewired I was a bit worried about this, as they state they don't support CRA2. But hey, it seems to work.